x/nft/keeper/msg_server_test.go

```diff
@@ -101,6 +101,7 @@ func (s *TestSuite) TestSend() {
 			_, err := s.nftKeeper.Send(s.ctx, tc.req)
 			if tc.expErr {
 				s.Require().Error(err)
+				fmt.Println(err.Error())
```

Comment:
Remove stray fmt.Println.

Reply:
will do!

Reply:
@faddat ser this is not you pr lol

Reply:
lol - thought this was one of mine :)  proceed